pkg/constants/metadata.go

```diff
@@ -36,6 +36,14 @@ const (
 	// see https://git-scm.com/docs/git-credential-store#_storage_format for more details
 	DevWorkspaceGitCredentialLabel = "controller.devfile.io/git-credential"
 
+	// DevWorkspaceGitTLSLabel is the label key to specify if the configmap is credentials for accessing a git server.
+	// Configmap must contain the following data:
+	// certificate: the certificate used to access the git server in Base64 ASCII
```

Comment:
Just wonder if:
- it makes sense to make it clear that it's private cert via configmap item.
- add an ability optionally to add public cert. Then users are able to grab it from there or dashboard/che server/theia could upload it to git provider. Probably it's already possible but let's make it clear